File: c++/30-chiziqli15.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;
int main()
{
	int x;
	double y,z,f;
	cin>>x;
	cin>>y>>z;
	f=pow(2,-x)*sqrt(x+pow(fabs(y)+2,1./4.))*pow(exp(x-1)/sin(z+2)+2,1./3.);
	cout<<fixed<<setprecision(2)<<f;
}

File: python/30-chiziqli15.py
import math
x,y,z=map(float, input().split())
f=pow(2,-int(x))*math.sqrt(int(x)+pow(abs(y)+2,1./4.))*pow(math.exp(int(x)-1)/math.sin(z+2)+2,1./3.)
print(format(f, '.2f')) 